index.bs

```diff
@@ -2038,7 +2038,7 @@ as that of another operation on the interface, however.
 This is how operation overloading is specified.
 
 The [=identifier=] of a [=static operation=]
-also must not be the same as the identifier
+may be the same as the identifier
```

Comment:
This paragraph used to be a normative restriction, but this change turns it into a clarification, so it should probably be a note.

Reply:
Don't use "may" in a non-normative note; probably "can" is better here. https://infra.spec.whatwg.org/#conformance